WINDOWS.md

```diff
@@ -557,9 +550,9 @@ It should look like this:
 To make sure everything works fine, close VS Code, then open your Ubuntu terminal:
 
 
->\- Press `Windows` + `R`
->\- Type  `ubuntu`
->\- Press `Enter`
+- Press `Windows` + `R`
```

Comment:
If we replace `>\` by normal markdown list, let's do it everywhere in every file to standardize the formatting. 

BTW, I understood why list are broken sometimes: this type of list require extra spaces at the end of each line but some text editor (my sublime for example) trim end-of-the-line spaces.

Reply:
Yes! Exactly what happens in VSC